pkg/util/util.go

```diff
@@ -39,3 +40,28 @@ func ValidateDriverName(driverName string) error {
 	}
 	return err
 }
+
+func CleanTargetPath(targetPath string) {
+	// remove the targetPath parent directory
+	// resolve kubelet issue https://github.com/kubernetes/kubernetes/issues/60987
+	_ = os.Remove(getParentDirectory(targetPath))
+}
+
+func CleanStagingTargetPath(stagingTargetPath string) {
+	// remove the stagingTargetPath parent directory
+	// resolve kubelet issue https://github.com/kubernetes/kubernetes/issues/60987
+	_ = os.Remove(getParentDirectory(stagingTargetPath))
+}
+
+func getParentDirectory(path string) string {
```

Comment:
use [path/filepath](https://golang.org/pkg/path/filepath/) instead of implementing your own